json/src/bytes.rs

```diff
@@ -33,38 +32,54 @@ impl Bytes {
     pub fn new(v: Vec<u8>) -> Self {
         Bytes(v)
     }
-}
 
-impl From<Bytes> for Vec<u8> {
```

Comment:
Is there any reason to moving `From<Bytes>` and `Deref` to below?

Reply:
I think these are miscellaneous, so these could be moved to below. If you prefer a smaller diff, I'll revert it.